modules/examiner/README.md

```diff
@@ -0,0 +1,82 @@
+# Examiner
+
+## Purpose
+
+The Examiner module displays the list of examiners in the study and allows 
+for additions and removal of certifications for each examiner listed.
+
+## Intended Users
+
+The primary type of users is:
+1. Site coordinators adding examiners and adding certifications to examiners
+
+
+## Scope
+
+This module displays information about examiners as well as certifications and 
+sites to which the examiners are affiliated. It also allows to add examiners 
+to sites, to give or remove certifications for instruments and set them as 
+radiologists. The Examiner architecture is completely separate from the User 
```

Comment:
The last sentence (2 sentences? Is it considered a separate sentence if there's a semicolon?) Should probably go under not in scope, since it's describing what it doesn't do.

Reply:
@driusan done